cmake/libprotobuf-lite.cmake

```diff
@@ -75,6 +75,6 @@ if(MSVC AND protobuf_BUILD_SHARED_LIBS)
 endif()
 set_target_properties(libprotobuf-lite PROPERTIES
     VERSION ${protobuf_VERSION}
-    OUTPUT_NAME ${LIB_PREFIX}protobuf-lite
-    DEBUG_POSTFIX "${protobuf_DEBUG_POSTFIX}")
-add_library(protobuf::libprotobuf-lite ALIAS libprotobuf-lite)
+    OUTPUT_NAME ${LIB_PREFIX}protobuf-lite${ver}
+    DEBUG_POSTFIX "${CMAKE_DEBUG_POSTFIX}")
```

Comment:
is there a strong reason to use `CMAKE_DEBUG_POSTFIX` instead of `protobuf_DEBUG_POSTFIX`? I'll probably be looking into why they have a `protobuf_DEBUG_POSTFIX` defined... why aren't they using the more generally available `CMAKE_DEBUG_POSTFIX`?

Reply:
I don't know if it is a strong reason, but it was the reason I used CMAKE_DEBUG_POSTFIX. On linux it didn't matter because protobuf_DEBUG_POSTFIX is always -d (which it is on linux). But when we build on windows, we have a -sd, not just -d. I figured my two options were either use CMAKE_DEBUG_POSTFIX or set protobuf_DEBUG_POSTFIX when we build it, but it would have to be conditional between windows and linux. 
Note: protobuf_DEBUG_POSTFIX is set in cmake/CMakeLists.txt. (don't know if that helps)